File: remote/src/main/java/com/kelsos/mbrc/utilities/FontUtils.java
package com.kelsos.mbrc.utilities;

import android.content.Context;
import android.graphics.Typeface;

public class FontUtils {

  public static Typeface getRobotoRegular(Context context) {
    return Typeface.createFromAsset(context.getAssets(), "fonts/roboto_regular.ttf");
  }

  public static Typeface getRobotoMedium(Context context) {
    return Typeface.createFromAsset(context.getAssets(), "fonts/roboto_medium.ttf");
  }
}

File: remote/src/main/kotlin/com/kelsos/mbrc/utilities/FontUtils.kt
package com.kelsos.mbrc.utilities

import android.content.Context
import android.graphics.Typeface

object FontUtils {

  fun getRobotoRegular(context: Context): Typeface {
    return Typeface.createFromAsset(context.assets, "fonts/roboto_regular.ttf")
  }

  fun getRobotoMedium(context: Context): Typeface {
    return Typeface.createFromAsset(context.assets, "fonts/roboto_medium.ttf")
  }
}
